extensions/amp-ad-network-adsense-impl/0.1/test/test-amp-ad-network-adsense-impl.js

```diff
@@ -90,6 +90,8 @@ describes.realWin('amp-ad-network-adsense-impl', {
     isResponsiveStub = sandbox.stub(impl, 'isResponsive_');
   });
 
+  afterEach(() => sandbox.restore());
```

Comment:
I think this test is somehow polluting the global test state. Does Travis still fail after removing this test code?

One guess is that `sandbox` variable here appears to be referencing a global -- restoring it unexpectedly may be related (generally you only need to restore sandboxes you create). Try using `env.sandbox` which is created by `describes.realWin`, scoped locally and does not require restoring.

Reply:
I added the restore earlier today to see if it would fix the issue, but I've tried with and without it, and it appears to make no difference. I'll try using env.sandbox.

Reply:
Hmm... no dice and I'm out of ideas. 

Try narrowing down the potential culprits by reverting more and more of this PR and see what happens. I'd start with temporarily removing all the test code.